docs/_docs/upgrading/3-to-4.md

```diff
@@ -46,6 +46,21 @@ you can invoke *`Liquid::Template`* directly:
   + template = Liquid::Template.parse(content)
   ```
 
+
+### Timezone in Windows
+
+Timezone utility for Jekyll on Windows now requires `tzinfo-2.0` and above.
```

Comment:
This sentence doesn't make a lot of sense to me. Perhaps `Timezone utilities` works here instead?

Reply:
It's in the singular because there's only one utility method for this in all of Jekyll: `Jekyll::Utils::WinTZ.calculate(timezone)`

Perhaps I should start the sentence with a definitive stance, then?
```diff
- Timezone utility for Jekyll on Windows now requires `tzinfo-2.0` and above.
+ The timezone utility for Jekyll on Windows now requires `tzinfo-2.0` and above.
```

Reply:
I'd rather be more generic and say `Timezone handling for Jekyll on Windows`...

It doesn't matter to the end user that there's only one. 🙂